icu4c/readme.html

```diff
@@ -23,8 +23,8 @@
         milestone - if the release note is a milestone release
 -->
 
-  <body>
-  <!-- <body class="rc"> -->
+  <!-- <body> -->
+  <body class="rc">
```

Comment:
I don't have a strong opinion either way, but I think in the BRS for 67 there was mention that we would go straight to the "final" readme instead of marking it as RC (if we don't plan on making changes to the readmes between releases), so that we wouldn't need another update for GA.

Reply:
You are right, we said that for 67: “just "67", not rc vs. final  updated instructions for this"”

However, I don't see the [instructions](https://sites.google.com/site/icusite/processes/release/tasks/publish#TOC-Update-readme) having been updated.

Leaving it up to @echeran whether he wants to merge as is and do a GA update later, or make another commit here to go straight to GA language.

Reply:
The instructions were updated with this: "If the readme should remain the same between milestones, we can skip directly to the GA description. Otherwise, pick the right one for the release type."

But I agree, either is fine. :)

Reply:
Done.  If had a reasoning for what you did for the last BRS, then I may as well be consistent here, so I also changed it to the "final" readme

Reply:
> The instructions were updated with this: "If the readme should remain the same between milestones, we can skip directly to the GA description. Otherwise, pick the right one for the release type."

I updated the instructions now. http://site.icu-project.org/processes/release/tasks/publish#TOC-Update-readme
Hopefully this is clearer going forward.